webhook_to_fedora_messaging_messages/github/__init__.py

```diff
@@ -0,0 +1,132 @@
+
+def summarize_repository_event(event_type: str, payload: dict) -> str:
+    
+    if event_type in ['push', 'fork']:
+        summary = _handle_repository_event(event_type, payload)
+    elif event_type.startswith('pull_request'):
+        summary = _handle_pull_request_event(event_type, payload)
+    elif event_type.startswith('issue'):
+        summary = _handle_issue_event(event_type, payload)
+    else:
+        summary = "Event type not supported"
+
+    return summary
+
+
+def _handle_repository_event(event_type: str, payload: dict) -> str:
```

Comment:
Maybe use the `summarize` prefix here as well instead of `handle` since it's generating a summary.